packages/zoe/test/unitTests/zcf/zcfSeat-exit.test.js

```diff
@@ -46,12 +46,11 @@ test(`zoe - wrongly throw zcfSeat.exit()`, async t => {
 
   // The contract uses the testJig so the contractFacet
   // is available here for testing purposes
-  /** @type {ContractFacet} */
+  /** @type {ZCF} */
   const zcf = testJig.zcf;
 
   /** @type {OfferHandler} */
   const throwSeatExit = seat => {
-    // @ts-expect-error Linting correctly identifies that exit takes no argument.
```

Comment:
`exit` actually takes an optional `Completion` which is any non-thenable Passable